### ISSUE TITLE
VOD Settings in management console is not working

### ISSUE THREAD

**Issue by mustafaboleken** [ant-media/Ant-Media-Server#5649]

**Comment by mekya** [comment]
Hi @mustafaboleken,

Could you please give the details?
Secondly, how do you feel when you receive a request in the support ticket by the same way?  
`VOD Settings in management console is not working`

I'm moving it to the backlog. After you give the details, I can set its status to `Next Sprint` again 

Regards
Oguz

**Comment by mustafaboleken** [comment]
@mekya 

You are right. To be honest, I created this one in a meeting and thought will modify later and forget it. It's my mistake.

**Comment by mekya** [comment]
We deprecate the `VoD Streaming Folder` and start to use `https://antmedia.io/rest/#/VoD%20Rest%20Service/importVoDs`

We can remove this settings from the webpanel.

**Comment by mustafaboleken** [comment]
Thats why it's not working. Okey, I will inform the customer.
